### PR TITLE
Add OPUS Wikimedia Translations Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/wikimedia_translations/README.md [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+
+[More Information Needed]
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/wikimedia_translations/README.md:352]: can you mention how to load the other languages like you did in other PRs please ?

File: datasets/wikimedia_translations/wikimedia_translations.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        super().__init__(
+            *args,
+            name=f"{lang1}-{lang2}",
+            **kwargs,
+        )
+        self.lang1 = lang1
+        self.lang2 = lang2
+
+
+class WikiMediaTranslations(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/wikimedia_translations/wikimedia_translations.py:70]:
```suggestion
class WikimediaTranslations(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `wikimedia_translations`

you'll have to regenerate the dataset_infos.json file after this change

**Review by lhoestq — COMMENTED** [review]
Nice :) 

one comment about the dataset class name :

**Comment by albertvillanova** [comment]
Thanks for your contribution, @abhishekkrthakur. Are you still interested in adding this dataset?

We are removing the dataset scripts from this GitHub repo and moving them to the Hugging Face Hub: https://huggingface.co/datasets

We would suggest you create this dataset there. Please, feel free to tell us if you need some help.

**Pull request closed without merging**